_data/betting.yml

```
@@ -5,3 +5,10 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
+
+    - name: win2day
+      url: https://www.win2day.at/
+      img: win2day.png
+      tfa: No
+      twitter: win2dayat
+      facebook: win2dayat
```

```suggestion
      facebook: win2dayat
      lang: de
      
```
Adding `lang: de` will translate the text on the call-to-action buttons. Additionally, the data files should end with a new line.